xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -1054,13 +1071,21 @@ final class DatabaseService {
             SELECT id, title
             FROM memo
             WHERE substr(memo.slug, 1, 1) != '_'
+                AND did = ?
```

Comment:
Restrict random selection to only `our` notes.

Reply:
Note this doesn't include draft content, which has a "fake" local data did.

In other SQL queries, we pass in, and then unpack a JSON array to match against any DID in the array.